Makefile

```
@@ -166,6 +166,33 @@ cerebro-fmt: ## Check Rust formatting for cerebro
 cerebro-build: ## Build cerebro binary
 	@cargo build --manifest-path modules/cerebro/Cargo.toml --release --bin cerebro
 
+# --- ROOK DISTRIBUTION FOUNDATIONS ---
+
+rook-dist-check: ## Validate Rook release artifact contract
+	@cargo test --manifest-path clients/rook/Cargo.toml --test distribution_contract
+
+rook-dist-local: ## Build current-platform Rook release artifact into clients/rook/dist
```

checkmake (0.3.2)

[warning] Target body for "rook-dist-local" exceeds allowed length of 5 lines (20).

(maxbodylength)